port/common/omrsysinfo_helpers.c

```diff
@@ -435,6 +435,35 @@ omrsysinfo_get_x86_description(struct OMRPortLibrary *portLibrary, OMRProcessorD
 	desc->features[3] = CPUInfo[CPUID_EBX]; /* Structured Extended Feature Flags in EBX */
 	desc->features[4] = CPUInfo[CPUID_ECX]; /* Structured Extended Feature Flags in ECX */
 
+	if (portLibrary->sysinfo_processor_has_feature(portLibrary, desc, OMR_FEATURE_X86_OSXSAVE)) {
+		/* Record extended processor state features from XCR0 */
```

Comment:
grammar: missing period
```suggestion
		/* Record extended processor state features from XCR0. */
```